### PR TITLE
[Celo Sage] 04-18-2023-how-to-store-NFT-smart-contract-assets-using-the-fileBase-decentralized-storage-system

### PR DESCRIPTION
**Pull request by johnfawole**
My tutorial follows the Tutorial Contribution Guidelines.
 - [x] The title of the Pull Request matches the title of my tutorial.
 - [x] I've followed the tutorial template structure.
 - [x] I've proofread the tutorial for spelling and grammar mistakes.
 - [x] I've thoroughly explained the code, with inline code comments and text explanations.
 - [x] I've run a linter & prettifier to format my code blocks.
 - [x] I've run the tutorial to test it and the code works.

Reviewers - @joenyzio, @viral-sangani

### CODE REVIEW

File: blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+title: How to Store NFT Smart Contract Assets Using The FileBase Decentralized Storage System
+description: This article gives a comprehensive overview of decentralized storage solutions. It contains a clear walk-through of how to store digital assets on the filebase storage solution.
+author:
```
Comment (Julius170) [inline blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md:4]: Add "s" to author and it should deploy properly

File: blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    url: https://www.linkedin.com/in/johnfawole/
+    image_url: https://github.com/johnfawole.png
+tags: ['solidity', 'smart contract', 'celosage', 'celo', 'filebase', 'decentralized storage']
+hide_table_of_contents: true
+slug: /tutorials/how-to-store-NFT-smart-contract-assets-using-the-fileBase-decentralized-storage-system
+---
+
+![filebase-cover](https://user-images.githubusercontent.com/105144630/232922465-f519c6cd-9e66-41aa-879e-e81b6c342f3c.png)
+
+# How to Store NFT Smart Contract Assets Using The FileBase Decentralized Storage System
```
Comment (Julius170) [inline blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md:16]: can you remove the title name here

File: blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+To mint the NFTs, you need to deploy the NFT contract. Once deployed, the contract owner can call the awardAttendee function passing in the receiver's address and the link to one of the NFTs stored on FileBase using the IPFS gateway URL.
+
+The recipient can verify that he owns an NFT on the blockchain and check for the image.
+
+## Conclusion
+FileBase is a web3 Storage-As-A-Service provider which provides a safer and more secure way to store information. It is an aggregation service that gives users access to multiple IPFS services, a feature that makes it the favorite. FiileBase is one of the best options to safeguard your NFT asset for the contract you deployed or from different NFT marketplaces.
+
+## NEXT STEP
+Filebase is not the only decentralized storage solution in Web3. Having gone through this tutorial, you can also check out Arweave and Filecooin. Have fun building.
```
Comment (Julius170) [inline blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md:140]: Can you add links to help the readers better, finding other related tutorials or videos on celo sage or any other platform

File: src/data-tutorials/users.tsx [hunk 4]
```diff
@@ -695,6 +695,15 @@ const Users: User[] = [
...
     tags: ['beginner','celosage','celo'],
   },
+  
+  
+    {
+    title: "How to Store NFT Smart Contract Assets Using The FileBase Decentralized Storage System",
+    description: "This article gives a comprehensive overview of decentralized storage solutions. It contains a clear walk-through of how to store digital assets on the filebase storage solution.",
+    preview: require('./showcase/intermediate/filebase-cover.png'),
+    website: '/tutorials/how-to-store-NFT-smart-contract-assets-using-the-fileBase-decentralized-storage-system',
+    tags: ['intermediate', 'solidity', 'smart contract', 'celosage', 'celo', 'filebase', 'decentralized storage'],
```
Comment (Julius170) [inline src/data-tutorials/users.tsx:705]: the tag "smart contract" shouldn't have a space in between It should be "smartcontract" same with "decentralizedstorage"

File: blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
+---
+title: How to Store NFT Smart Contract Assets Using The FileBase Decentralized Storage System
+description: This article gives a comprehensive overview of decentralized storage solutions. It contains a clear walk-through of how to store digital assets on the filebase storage solution.
+author:
+  - name: John Fawole
+    title: Technical Writer
+    url: https://www.linkedin.com/in/johnfawole/
+    image_url: https://github.com/johnfawole.png
+tags: ['solidity', 'smart contract', 'celosage', 'celo', 'filebase', 'decentralized storage']
```
Comment (Julius170) [inline blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md:9]: make the same changes suggested in the user.tsx file's tags here

File: blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+hide_table_of_contents: true
+slug: /tutorials/how-to-store-NFT-smart-contract-assets-using-the-fileBase-decentralized-storage-system
+---
+
+![filebase-cover](https://user-images.githubusercontent.com/105144630/232922465-f519c6cd-9e66-41aa-879e-e81b6c342f3c.png)
```
Comment (Julius170) [inline blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md:14]: can you reference the header image from the user.tsx file, the one you uploaded, 
---according to the latest quidelines on the community

**Review by Julius170 — COMMENTED** [review]
Hi @johnfawole Ive made some suggestions, Once you've addressed them, your good to go 🥂

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**